app = App#14:516{owner:13_1154,mng:17_0,versions:[105],dslPackage:flow2b.xero,name:,description:,uuid:189523713439291072855282057135459059560} v3
dslPackage = flow2b.xero
releaseNotes = <p>process Xero Internal Server error(500)</p><p><br></p><p>B4176&nbsp;SO not showing in Xero</p><p><span data-auto-link="true" data-href="https://flow2b.com/s/#13_1/flow2b.net.dev.Bug/form/15_29918"><a href="https://flow2b.com/s/#13_1/flow2b.net.dev.Bug/form/15_29918">https://flow2b.com/s/#13_1/flow2b.net.dev.Bug/form/15_29918</a></span></p>
@rid = #12:2314
touchedOn = Sat Feb 23 07:48:20 UTC 2019
version = 5.18.8
uuid = 7010642943870366714938683487382535818
dependencies = [{dslPackage:flow2b.inv.order,version:[11.0.0, 12.0.0),channel:Pretest}, {dslPackage:flow2b.integ,version:,channel:Pretest}, {dslPackage:flow2b.acc.asset,version:,channel:Pretest}]
status = Canceled
